apps/browser/settings/searchenginemodel.cpp

```diff
@@ -25,6 +26,13 @@ SearchEngineModel::SearchEngineModel(QObject *parent)
         SearchEngine engine(QUrl(), name, true);
         m_searchEngines.append(engine);
     }
+
+    MGConfItem gconf(SEARCH_ENGINES_AVAILABLE_CONFIG);
```

Comment:
Wondering if this should be member variable as allocated 3 times when search service is not yet installed.